src/status_im/ui2/screens/chat/composer/input.cljs

```diff
@@ -168,12 +174,130 @@
       :on-selection-change      (partial on-selection-change timeout-id last-text-change mentionable-users)
       :on-change                (partial on-change last-text-change timeout-id mentionable-users refs chat-id sending-image)
       :on-text-input            (partial on-text-input mentionable-users chat-id)}
-     (if mentions-enabled
-       (for [[idx [type text]] (map-indexed
-                                (fn [idx item]
-                                  [idx item])
-                                (<sub [:chat/input-with-mentions]))]
-         ^{:key (str idx "_" type "_" text)}
-         [rn/text (when (= type :mention) {:style {:color colors/primary-50}})
-          text])
-       (get @input-texts chat-id))]))
+     (fn []
+       (if mentions-enabled
+         (for [[idx [type text]] (map-indexed
+                                  (fn [idx item]
+                                    [idx item])
+                                  (<sub [:chat/input-with-mentions]))]
```

Comment:
Please change `<sub` to `rf/sub` in this PR and future PRs. Also, for dispatching events we will be using `rf/dispatch`.